src/components/Panel.tsx

```diff
@@ -72,22 +72,20 @@ const Panel: React.FC<PanelProps> = (props) => {
   const firstRender = useRef(true)
   const [store, dispatch] = useStore((store) => ({
     selectedComment: store.selectedComment,
-    selectedSuggestion: store.selectedSuggestion,
-    editorSelectedSuggestion: store.editorSelectedSuggestion,
```

Comment:
Another mention of 'editorSelectedSuggestion' in line 139 needs to be removed.

Reply:
Removed.